backend/lib/spree/backend_configuration.rb

```diff
@@ -6,6 +6,27 @@ module Spree
   class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: I18n.default_locale
 
+    # @!attribute [rw] themes
+    #   @return [Hash] A hash containing the themes that are available for the admin panel
+    preference :themes, :hash, default: {
+      classic: 'spree/backend/all',
+    }
+
+    # @!attribute [rw] theme
+    #   @return [String] Default admin theme name
+    preference :theme, :string, default: nil
+
+    def default_theme
+      :classic
+    end
+
+    def theme_path(user_theme = nil)
+      theme_path = themes.fetch(user_theme.to_sym, nil) if user_theme
+      theme_path ||= themes.fetch(theme.to_sym, nil) if theme
+      theme_path ||= themes.fetch(default_theme, nil)
+      theme_path
+    end
```

Comment:
I'm probably missing something, but it looks like too complicated logic. Shouldn't we have a `default: :classic` for the `theme` preference, and default the `user_theme` argument in the `theme_path` function to the `theme` preference value? :thinking:

Reply:
@waiting-for-dev maybe I was too defensive here, I can remove some of the fallback layers and expect that if you set a theme that theme is available (maybe raise an exception via `fetch` if it's not).